### PR TITLE
Update experiments doc

### PR DESCRIPTION
**Pull request by zequnyu**
#978.

### CODE REVIEW

File: docs/user/experiments.rst [hunk 1]
```diff
@@ -10,43 +10,40 @@ We use object oriented abstractions for different components required for an exp
...
 
-    import gym
-
     from garage.experiment import run_experiment
     from garage.np.baselines import LinearFeatureBaseline
     from garage.tf.algos import TRPO
     from garage.tf.envs import TfEnv
-    from garage.tf.policies import CategoricalMLPPolicy
+    from garage.tf.experiment import LocalTFRunner
+    from garage.tf.policies import
```
Comment (gitanshu) [inline docs/user/experiments.rst]: Looks like the import got missed
```suggestion
    from garage.tf.policies import CategoricalMLPPolicy
```

File: docs/user/experiments.rst [hunk 2]
```diff
@@ -10,43 +10,40 @@ We use object oriented abstractions for different components required for an exp
...
+
 
+    def run_task(snapshot_config, *_):
+        """Run task."""
```
Comment (gitanshu) [inline docs/user/experiments.rst]: I think the existing docstring was fine
```suggestion
        """Wrap TRPO training task in the run_task function."""
```

File: docs/user/experiments.rst [hunk 3]
```diff
@@ -119,24 +116,17 @@ Additional arguments for `run_experiment`:
 Running Experiments with TensorFlow and GPU
 ===========================================
 
-To run experiments in the TensorFlow tree of garage with the GPU enabled, set the flags use_tf and use_gpu to True when calling `run_experiment`, as shown in the code below:
+Using `tensorflow-gpu`, TensorFlow has GPU enabled by default. Set the flags `force_cpu` to True to disable GPU, as shown in the code below:
```
Comment (gitanshu) [inline docs/user/experiments.rst]: Maybe a nitpick, but I suggest this to be consistent with installation docs
```suggestion
If you installed the `garage[gpu]` extra package, TensorFlow will use GPU by default. To disable GPU, set the flag `force_cpu` to True, as shown in the code below:
```

Comment (zequnyu) [reply docs/user/experiments.rst]: Thanks for suggestions! Updated.

**Pull request merged**